src/components/JournalEntriesView.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Dispatch } from 'react';
+import { connect } from 'react-redux';
+import { Action } from 'redux';
+import { createStyles, WithStyles, withStyles } from '@mui/styles';
+import { TdrState } from 'reducers';
+import { JournalEntryModel, IamResourceTypeEnum } from 'generated/tdr';
+import { getJournalEntries } from 'actions/index';
+import JournalEntryTable from './table/JournalEntryTable';
+
+const styles = () =>
+  createStyles({
+    className: {
+      color: 'red',
+    },
+  } as const);
+
+interface JournalEntriesViewProps extends WithStyles<typeof styles> {
+  id?: string;
+  resourceType?: IamResourceTypeEnum;
+  journalEntries: JournalEntryModel[];
+  loading: boolean;
+  refreshCnt: number;
+  dispatch: Dispatch<Action>;
+}
+
+function JournalEntriesView({
+  id,
+  resourceType,
+  journalEntries,
+  loading,
+  refreshCnt,
+  dispatch,
+}: JournalEntriesViewProps) {
+  const handleFilterJournalEntries = (limit: number, offset: number) => {
+    dispatch(getJournalEntries({ id, resourceType, limit, offset }));
+  };
+
+  return (
+    <JournalEntryTable
+      journalEntries={journalEntries}
+      handleFilterJournalEntries={handleFilterJournalEntries}
+      loading={loading}
+      refreshCnt={refreshCnt}
+    />
```

Comment:
Why is JournalEntryTable in a separate component if it is all that is returned by JournalEntriesView?

Reply:
That's a good question! I was following the design that we did for DatasetView -> DatasetTable, to try to keep it the same. The wrapper (JournalEntriesView) extracts the state variables and passes them in to the JournalEntryTable. Do you think we should merge them?

Reply:
Gotcha! That's fine to leave it as is. Thanks for explaining!